### DISCUSSION TITLE
Cooperative vs. Nonprofit

### DISCUSSION THREAD

**Discussion by seekanddefine** [knowledge-for-good/knowledgeforgood#20] [Q&A]
Should we be a nonprofit or a cooperative corporation? We can also adopt cooperative principles to manage the nonprofit but there are pros and cons of being registered as either. Both exist to benefit their constituencies rather than shareholders and both can be governed democratically. Primarily the legal distinction is one of how money flows. In a co-op you typically see a distribution of profits based on participation and in a nonprofit there are more limitations such that profits cannot. Another distinction is that a co-op typically generates most or all of its revenue through the sale of goods and services, which a nonprofit is limited in the amount of business activity it conducts unrelated to a charitable purpose.

**Comment by seekanddefine** [comment]
https://www.wired.co.uk/article/the-tech-cooperatives-changing-the-way-startups-do-business

Co-ops and blockchain can change how startups do business

Q1: Should every employee be paid the same?

A1:Research tech co-op networks/supports
-There are some co-ops working together to found collective marketing groups with cooperative values
-start-up accelerators may also provide valuable resources, mentor pairing, crowdfunding session to raise seed funding
-cooperative banks
-using "community shares": can't be sold for-profit, withdrawable and upon which fixed interest in profitable years
-"sweaty equity" recording volunteer hours to be paid back later

Q2: Does sharing more fairly provide better social responsibility?

Q3: What is the difference between democratic voting and sociocratic voting?
-Sociocratic voting was invented by the Quakers, agreement is reached by consensus rather than majority

Q4: Should all staff be "employee-partners"?
-vote for members of the board
-entitled to dividends
-Another option is the consumer co-op where shoppers vote on key issues and vote for certain elected representatives
-Could also consider interconnected co-ops run independently under a central organization

Q5: What resources could we offer the entire community as we learn and grow to document and be transparent?
-Trainings, Monthly meet-ups to hold discussions/panels, share skills

"the tech world's newfound collectivist side"
Q6: How do we build community in a way that shows we are part of this movement and support others in it?
-snowball effect


Robbins on why co=ops are better than private businesses:
"“We run our organisations to run good services. Other organisations make money by providing good services. We're genuinely trying to solve the problems and make things better.”"

Q6: Should we adopt a "Robin Hood model"?
-won't work with petrochemical companies, arms manufacturers, or the sex trade
-collaborates with government agencies and financial services firms
-surplus accrued after paying salaries is either donated directly of allocated to time spent developing pro bono projects

We are committed to grow with a purpose that's socially and ethically-minded.

**Reply by avidrucker** [reply]
I'd love to convert this to a list in order to explore/upvote/downvote individual ideas.

**Comment by seekanddefine** [comment]
https://ncbaclusa.coop/resources/7-cooperative-principles/

7 Cooperative Principles:

THE 7 COOPERATIVE PRINCIPLES

1. VOLUNTARY & OPEN MEMBERSHIP
Anyone can join a co-op—they don’t discriminate based on gender, social, racial, political or religious factors.


2. DEMOCRATIC MEMBER CONTROL
Members control their business by deciding how it’s run and who leads it.


3. MEMBERS' ECONOMIC PARTICIPATION
All co-op members invest in their cooperative. This means people, not shareholders, benefit from a co-op’s profits.


4. AUTONOMY & INDEPENDENCE
When making business deals or raising money, co-ops never compromise their autonomy or democratic member control.


5. EDUCATION, TRAINING AND INFORMATION
Co-ops provide education, training and information so their members can contribute effectively to the success of their co-op.


6. COOPERATION AMONG COOPERATIVES
Co-ops believe working together is the best strategy to empower their members and build a stronger co-op economy.


7. CONCERN FOR COMMUNITY
Co-ops are community-minded. They contribute to the sustainable development of their communities by sourcing and investing locally.

**Comment by seekanddefine** [comment]
https://www.coops.tech/

Creative Cooperative Technology Companies Network UK

"All the members of CoTech are available to hire, and each has different specialisms. We vary in size from 2 people to 40+ people, and use different technologies. If you know what technology or service you're looking, you can use this site to find suitable co-ops. If you're not sure what you're looking for you can browse the members and get in touch with one that you like the look of. We're all committed to being honest about our capabilites and capacities, and we'll refer you to a different co-op if we think they'll offer you better value.


If you don't want to hire us but want to find out more about co-ops, find a job in a co-op or discuss how technology can help solve the world's myriad problems, please head over to our Discourse forum."

**Reply by seekanddefine** [reply]
Discourse forum https://community.coops.tech/

**Comment by seekanddefine** [comment]
https://github.com/hng/tech-coops

A list of tech coops and resources concerning tech coops and worker owned cooperatives in general.

**Reply by seekanddefine** [reply]
There isn't a single education co-op on this worldwide list.

**Reply by avidrucker** [reply]
> There isn't a single education co-op on this worldwide list.

😲

**Comment by seekanddefine** [comment]
https://github.com/mttmyr/awesome-tech-cooperatives

The awesome list which is much shorter than the tech-coop list.

**Comment by seekanddefine** [comment]
Do you think it would be a good idea to support certain kinds of business first when we are purchasing or using goods or services?

Which would become a greater priority? 
-Opensource
-Cooperative
-Diverse
-Accessible

**Reply by avidrucker** [reply]
Off hand, I'm not sure how this would go into consideration, without further context. I think it'd be great to work with like-minded individuals in the sense that, we can learn a lot from their expertise and wisdom. On the other hand, restricting our business to those who meet certain criteria I think would warrant, at least for me, a reflection upon why. I strongly agree with your statement to not support human trafficking, for example.

**Comment by seekanddefine** [comment]
Source: 30p [Digital Democracy? Options for the International Cooperative Alliance to advance Platform Coops](The practice of NVC entails an intention to connect compassionately with ourselves and with others, and an ability to keep our attention in the present moment – which includes being aware that sometimes in this present moment we are recalling the past, or imagining a future possibility.)

>platform coops: “an enterprise that operates primarily through digital platforms for
interaction or the exchange of goods and/or services and is structured in line with the International
Cooperative Alliance Statement on the Cooperative Identity.”
>
>Four key barriers: 
>1. Governance: Good governance in terms of design and practice is important to all
cooperatives and there are likely to be distinctive challenges, opportunities and points of
learning across platform coops in the absence of a geographically-rooted community
>2. Technology: Platform cooperatives may be limited in adopting the technologies that
underpin Big Tech, for both practical and principled reasons. For existing coops, including
some platform coops, the technology is not always released on an open source basis,
limiting the potential for wider collaboration.
>3. Growth: Platform cooperatives are unlikely to follow the established growth strategies of
incumbents and so will find it difficult to build the natural “network effects” that fuel market
penetration and consolidation.
>4. Capital: Start-up costs may be small, but scaling up, both in terms of customer acquisition
and in terms of spreading operations across legal and regulatory jurisdictions, is resource-
intensive. Platform cooperatives will struggle to access the capital they need to grow given
the entrenched financing landscape in the start-up tech sector predicated on venture
capital.
>
>See Appendix: Typology list of online platforms

**Reply by avidrucker** [reply]
I would like to a brainstorming session on the challenges we face today in education, online education, adult education, and in technical education, as well as, challenges for co-ops, and our current team's challenges. This will help us, I believe, start drafting some strategies to move forward with confidence and enthusiasm :)

**Reply by seekanddefine** [reply]
@avidrucker put a time on your calendar for tomorrow. Let me know if it works. Hoping to focus on the coop/nonprofit discussion.

**Comment by seekanddefine** [comment]
[Github provides discounts for nonprofits](https://docs.github.com/en/billing/managing-billing-for-your-github-account/discounted-subscriptions-for-github-accounts)

Apply for a discount on the [Social Impact page](https://socialimpact.github.com/).
